pkg/utils/deleterelayservice.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func DeleteRelayAgent(kubeConfig []byte, namespace string) bool {
+
+	kubeconfig := flag.String("kubeconfig", string(kubeConfig[:]), "kubeconfig file yaml byte")
```

Comment:
We will not be passing any flags to paralus, so this should not be there.

flag.String essentially expects a --kubeconfig flag while running paralus otherwise defaults with the second value..